aiohttp/web_urldispatcher.py

```diff
@@ -296,7 +264,11 @@ def get_info(self):
         return {'path': self._path}
 
     def url(self, *, query=None):
-        return self._append_query(self._path, query)
+        super().url()
```

Comment:
`url_for` returns `URL`, not `str`.
But functionally it's a replacement for deprecated API.